TASKS/Program.cs

```diff
@@ -9,7 +9,17 @@ namespace TASKS
     internal class Program
     {
         static void Main(string[] args)
-        {
+        {   Console.Write("Введите радиус круга: ");
+            double radius = Convert.ToDouble(Console.ReadLine());
+
+            double circumference = 2 * Math.PI * radius;
+            double area = Math.PI * Math.Pow(radius, 2);  
```

Comment:
Квадрат числа проще вычислить через умножение:
```suggestion
            double area = Math.PI * radius * radius;  
```